src/components/editor-pane-container.jsx

```diff
@@ -53,15 +54,15 @@ class EditorPaneContainer extends React.Component {
           }}
           handleClasses={{ right: 'resizer' }}
           bounds="window"
-          minWidth={300}
+          minWidth={this.props.paneMinWidth}
           onResizeStop={this.onResizeStopHandler}
           size={{ width: this.props.cellResizerWidth }}
           style={{
             display: this.props.cellResizerDisplayStyle,
             flexDirection: 'column',
           }}
         >
-          <div id="cells">
+          <div style={this.props.cellsStyle} id="cells">
```

Comment:
@bcolloran do you have any thoughts on this approach?

Reply:
@hamilton seems reasonable enough to me, do you have any particular concern?